### PR TITLE
Fix rest of linting problems

### PR DESCRIPTION
**Pull request by rejas**
Fixed the remaining linting problems after https://github.com/deg0nz/MMM-PublicTransportBerlin/pull/77

now really based against develop :-)

### CODE REVIEW

File: .stylelintrc.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
 	"root": true,
   "rules": {
     "prettier/prettier": true,
-    "no-eol-whitespace": true
+    "no-eol-whitespace": true,
+    "selector-class-pattern": null
```
Comment (KristjanESPERANTO) [inline .stylelintrc.json:8]: I see the disadvantage that custom styles might not work anymore, but nonetheless I would like to transform the names to kebab-case. Then we would not need to disable this rule. What do you think?

File: BvgFetcher.js [hunk 2]
```diff
@@ -90,9 +89,12 @@ class BvgFetcher {
       // TODO: Make real stop/station handling here
       // Quick fix to work around missing station objects
       if (!row.station) {
-        row.station = row.stop;
+        row.station = row.stop; // eslint-disable-line no-param-reassign
       }
 
+      // Set hidden debug option to print debug info about departures
+      if (this.config.debug) this.printDeparture(row);
```
[the remark this answers is not in the document]
Comment (rejas) [reply BvgFetcher.js]: Changed it to general config

File: .stylelintrc.json (hunk 1 above)
Comment (rejas) [reply .stylelintrc.json:8]: Maybe doing the transformation should be done in another PR after this is merged (and when @deg0nz weighted in on this)?

**Review by KristjanESPERANTO — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**